src/components/common/SearchInput/ExperimentalVault.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Container, TextField, Button, Grid } from '@material-ui/core';
+import styled from 'styled-components';
+
+const StyledForm = styled.div`
+    && {
+        width: 100%;
+        align-items: center;
+    }
+`;
+const StyledContainer = styled(Container)`
+    && {
+        width: 100%;
+        align-items: center;
+        align-content: center;
+        background-color: ${({ theme }) => theme.container} !important;
+        border-radius: 8px;
+    }
+`;
+const StyledTextField = styled(TextField)`
+    && {
+        width: 100%;
+        background-color: transparent;
+        align-content: center;
+        border-color: transparent;
+        .MuiOutlinedInput-root {
+            fieldset {
+                border-color: transparent !important;
+            }
+            &:hover fieldset {
+                border-color: transparent !important;
+            }
+            &.Mui-focused fieldset {
+                border-color: transparent;
+            }
+            color: ${({ theme }) => theme.text} !important;
+        }
+    }
+`;
+const StyledHeader = styled.div`
+    && {
+        color: #363537;
+        margin-left: 1em;
+    }
+`;
+
+export const ExperimentalVault = () => {
+    const [contract, setContract] = useState<string>('');
+    const history = useHistory();
+
+    const handleClick = (event: React.ChangeEvent<any>) => {
+        history.push(`/vault/${contract}`);
```

Comment:
since the PR started we now support networks like arbitrum and fantom on URL, so this would default to ethereum network but we may want to handle other networks too